packages/design-system/src/types/MonthPicker/MonthPicker.d.ts

```diff
@@ -28,6 +30,14 @@ export interface MonthPickerProps {
    */
   label: React.ReactNode;
   errorMessage?: React.ReactNode;
+  /**
+   * Additional classes to be added to the error message
+   */
+  errorMessageClassName?: string;
+  /**
+   * Location of the error message relative to the field input
+   */
+  errorPlacement?: MonthPickerErrorPlacement;
```

Comment:
For future reference @nichia  you dont need to define a new type here, you can just use the inline `"top" | "bottom"` value.  Its only setup this way because we used a script to auto generate these typings initially